### PR TITLE
fix: isConnected returning true for not connected accounts

### LINKED ISSUE #929: isConnected returning true even if currentAccount is not connected
When completing this task, fix it here according to the comment: 
useBridgeTxs line 43. // TODO

### PR DESCRIPTION
**Pull request by matt-user**
closes #929

### CODE REVIEW

File: packages/app/src/systems/CRX/background/services/BackgroundService.ts [hunk 1]
```diff
@@ -181,7 +181,13 @@ export class BackgroundService {
...
 
   async isConnected(_: unknown, serverParams: EventOrigin) {
-    return !!serverParams.connection;
+    const currentAccount = await AccountService.getCurrentAccount();
+    if (!serverParams.connection) {
+      return false;
+    }
+    return serverParams.connection.accounts.includes(
+      currentAccount?.address || ''
+    );
```
Comment (luizstacio) [inline packages/app/src/systems/CRX/background/services/BackgroundService.ts:190]: We should change this implementation to add an optional param to isConnected.

The idea of the isConnected method is not connected to the current account but to the state of the connection.

Another way to know if an account is connected or not is by checking on the accounts list.

**Review by luizstacio — CHANGES_REQUESTED** [review]

File: packages/app/src/systems/CRX/background/services/BackgroundService.ts (hunk 1 above)
Comment (luizstacio) [reply packages/app/src/systems/CRX/background/services/BackgroundService.ts:190]: The current behavior is not a bug but an expected behavior. cc: @LuizAsFight

Comment (LuizAsFight) [reply packages/app/src/systems/CRX/background/services/BackgroundService.ts:190]: what's the benefit of knowing that the account is connected but you can't do nothing as the current account is not connected? on this `isConnected` has no use as you can't trust it to show "connect" button for example.

Comment (luizstacio) [reply packages/app/src/systems/CRX/background/services/BackgroundService.ts:190]: You can request a signature or send transactions to any connected account.

Comment (LuizAsFight) [reply packages/app/src/systems/CRX/background/services/BackgroundService.ts:190]: @matt-user we can close this PR then

Comment (matt-user) [reply packages/app/src/systems/CRX/background/services/BackgroundService.ts:190]: @LuizAsFight ok I'll close this PR and refactor the code on the bridge

**Comment by matt-user** [comment]
closed as this is not a bug, but a feature

**Pull request closed without merging**